robot_smach_states/src/robot_smach_states/perception.py

```diff
@@ -88,6 +100,51 @@ def run(self, robot, entity, area, waittime):
         return "failed"
 
 
+class RotateToEntity(smach.State):
```

Comment:
Migrate to navigation

Reply:
I would leave it here. As the LookAtXXX states are also here.